client/source/DelphiLint.LogViewer.dfm

```diff
@@ -0,0 +1,272 @@
+object LogViewerForm: TLogViewerForm
+  Left = 0
+  Top = 0
+  Caption = 'Analysis Log'
+  ClientHeight = 434
+  ClientWidth = 892
+  Color = clBtnFace
+  Constraints.MinHeight = 300
+  Constraints.MinWidth = 755
+  Font.Charset = DEFAULT_CHARSET
+  Font.Color = clWindowText
+  Font.Height = -12
+  Font.Name = 'Segoe UI'
+  Font.Style = []
+  FormStyle = fsStayOnTop
+  TextHeight = 15
+  object BottomPanel: TPanel
+    Left = 0
+    Top = 389
+    Width = 892
+    Height = 45
+    Align = alBottom
+    TabOrder = 0
+    object Label3: TLabel
+      Left = 8
+      Top = 6
+      Width = 718
+      Height = 15
+      Caption = 
+        'Please note that errors or warnings during an analysis can affect ' +
+        'the quality of the results, including false positive or false negative ' +
+        'issues.'
```

Comment:
Is this a bit verbose? I'd have probably written it in a much more terse fashion like below, though I'm conscious that you want a consistent tone in the DelphiLint hints/messages.
```
Errors/warnings may affect analysis quality, causing missing issues or false positives.
```